client/dashboard/me/billing-purchases/dataviews.tsx

```diff
@@ -175,6 +181,30 @@ function OwnerInfo( {
 	return <BillingPurchaseInfoPopover>{ tooltipContent }</BillingPurchaseInfoPopover>;
 }
 
+export function PurchaseSettingLink( {
```

Comment:
Are there other places you expect to use this? If so, perhaps we should move it to a higher level directory?